src/sentry.ts

```diff
@@ -172,8 +172,11 @@ export default class SentryService {
         const filtered_sentry_args = await this.filterNonSensitiveSentryMetadata(non_sensitive, args);
         const filtered_sentry_flags = await this.filterNonSensitiveSentryMetadata(non_sensitive, flags);
 
-        await this.setScopeExtra('command_args', filtered_sentry_args);
-        await this.setScopeExtra('command_flags', filtered_sentry_flags);
+        await this.setScopeExtra('args', filtered_sentry_args);
```

Comment:
Move this tag setting logic to start transaction so we don't set info in two different places

```ts
const { args, flags, metadata } = await this.command.parse(command_class);

const argv_flags = Object.entries(metadata.flags)
  .filter(([flag, metadata]) => !metadata.setFromDefault)
  .map(([flag, metadata]) => flag)
 ```

Reply:
moved setting scope/data/tags into the start transaction.